pyuvdata/uvcal/calfits.py

```diff
@@ -61,8 +61,8 @@ def write_calfits(
         ):
             warnings.warn(
                 "The calfits format does not support recording optional phase center, "
-                " scan number, or time-varying reference antenna information, and will "
-                "be dropped."
+                "scan number, or time-varying reference antenna information, and "
+                "these information will be included in the written file."
```

Comment:
will be included? or will not be included?

Reply:
Well that's just embarrassing -- should  be "not included". Fixed in 70f7fed6b2f12e6dfd1aff1c9e18d1627cd8c8b9.